virtcontainers/device/drivers/utils.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2017-2018 Intel Corporation
+// Copyright (c) 2018 Huawei Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package drivers
+
+import (
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/kata-containers/runtime/virtcontainers/device/api"
+)
+
+func deviceLogger() *logrus.Entry {
+	return api.DeviceLogger()
+}
+
+// FIXME: this is duplicate code from virtcontainers/hypervisor.go
+const maxDevIDSize = 31
+
+// FIXME: this is duplicate code from virtcontainers/hypervisor.go
```

Comment:
Sorry; why can't we remove this from hypervisor.go @WeiZhang555

Reply:
Because my original idea is we don't expose `virtcontainers/device/drivers` package directly to hypervisor at all (or at least not expose too much), and I don't want to let hypervisor.go invoke a `drivers.MakeNameID()` call, that's why I didn't remove it from hypervisor.go. 

Maybe we can find a better way to handle these common functions with more refactor and further works, currently I don't have a certain thought for this.